lib/order.rb

```diff
@@ -0,0 +1,61 @@
+# I would like to be able to select some number of several available dishes
+# I would like to check that the total I have been given matches the sum of the various dishes in my order
+
+require_relative 'menu.rb'
+
+class Order
+
+attr_reader :user_order, :place_order, :quantity
+
+  def initialize
+    @user_order = []
+    @basket = Hash.new([0.0])
+    @menu = Menu.new
+  end
+
+  # def basket
+  #   @basket.dup
+  # end
+  #this works but query best for price?
+
+  def price(item)
+  fail 'Item not in menu!' unless !!@menu_contents.include?(item)
```

Comment:
Style/DoubleNegation: Avoid the use of double negation (`!!`). (https://github.com/bbatsov/ruby-style-guide#no-bang-bang)<br>Style/NegatedIf: Favor `if` over `unless` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)